cmd/kubeadm/app/discovery/token/token.go

```diff
@@ -208,7 +208,7 @@ func getClusterInfo(client clientset.Interface, kubeconfig *clientcmdapi.Config,
 	}
 
 	klog.V(1).Infof("[discovery] Waiting for the cluster-info ConfigMap to receive a JWS signature"+
-		"for token ID %q", token.ID)
+		" for token ID %q", token.ID)
```

Comment:
Do we use `fmt.sprintf` to optimize this hard-coded style?
```go
msg := fmt.Sprintf("[discovery] Waiting for the cluster-info ConfigMap to receive a JWS signature for token ID %q", token.ID)
klog.V(1).Infof(msg)
```

Reply:
agree